x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.tsx

```diff
@@ -94,7 +109,7 @@ const getDefaultFieldTableColumns = ({
           </EuiFlexItem>
 
           <EuiFlexItem grow={false}>
-            <FieldName fieldId={name} highlight={highlight} />
+            <FieldName fieldId={getFieldItemName(name)} highlight={highlight} />
```

Comment:
I've seen that when searching `cases` or `Case` in the browser field modal the column doesn't shows up, it's still searching by the column id instead of the label (maybe it should search through both)

![Screenshot 2](https://user-images.githubusercontent.com/17549662/223056047-144c457a-f8a6-4135-a171-dabd51ddf36b.png)

Reply:
Yea, I saw this same issue when pulling down, not sure what the expected behavior is

Reply:
I think the only way to make it work at this time will be to change this condition in this files `x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/helpers.ts` at L62.

Note the cleanest way of doing it and we can create a ticket to add a new property in the `browserfields` to search on alternate value.

#### old code
```ts
      // Check if this field matches (via substring comparison) the passed substring
      if (fieldNameFromDescriptor !== null && fieldNameFromDescriptor.includes(trimmedSubstring)) {
        // this field is a match, so we should emit this category into the result object.
        hadAMatch = true;

        // emit this field
        filteredFields[fieldName] = fieldDescriptor;
      }
```

#### new code
```ts
      // Check if this field matches (via substring comparison) the passed substring
      if (
        fieldNameFromDescriptor !== null &&
        (fieldNameFromDescriptor.includes(trimmedSubstring) ||
          (fieldNameFromDescriptor === ALERT_CASE_IDS &&
            'cases'.includes(trimmedSubstring.toLocaleLowerCase())))
      ) {
        // this field is a match, so we should emit this category into the result object.
        hadAMatch = true;

        // emit this field
        filteredFields[fieldName] = fieldDescriptor;
      }
```

Reply:
We should use a translations string instead of hard coded value of `cases`

Reply:
@michaelolo24 @XavierM I fixed it. Could you please take another look?